packages/data-transport-layer/src/services/server/service.ts

```diff
@@ -157,8 +158,15 @@ export class L1TransportServer extends BaseService<L1TransportServerOptions> {
       'get',
       '/eth/syncing',
       async (): Promise<SyncingResponse> => {
+        this.logger.info('Retrieving L2 syncing status...')
```

Comment:
These loglines will happen with each API request, there is already a logline per API request. I recommend a single change here:

https://github.com/ethereum-optimism/optimism/blob/8ae059c64114467ab085183ebd40095595780f3e/packages/data-transport-layer/src/services/server/service.ts#L129

Adding `body: json`

Then we can get rid of all of these logs so that we maintain 1 logline per API request

Reply:
ooo MUCH better, def should've caught this!